src/shared_modules/utils/windowsHelper.h

```diff
@@ -141,6 +144,80 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(std::string timestamp1, std::string timestamp2)
+    {
+        if (timestamp1.size() == 8)
+        {
+            std::string::iterator it;
+
+            for (it = timestamp1.begin(); it != timestamp1.end(); ++it)
```

Comment:
This code portion could be also a util function.
We have some examples in https://stackoverflow.com/questions/8888748/how-to-check-if-given-c-string-or-char-contains-only-digits

like 

```c++
bool is_digits(const std::string &str)
{
    return str.find_first_not_of("0123456789") == std::string::npos;
}
```

Reply:
Moved that function to stringHelper.h